src/Form/LegacyConsumer/Commands/SetupFieldConfirmation.php

```diff
@@ -32,20 +32,20 @@ public function __invoke( $hook ) {
 
 		$formID = give_get_payment_meta( $this->payment->ID, '_give_payment_form_id' );
 
-		$fieldCollection = new FieldCollection( 'root' );
-		do_action( "give_fields_{$hook}", $fieldCollection, $formID );
+		$collection = Group::make( 'root' );
```

Comment:
@knowler Does this mean we're making one Group per hook that has a name of "root"? I just want to make sure we're not getting into name collision in the future when these are all merged into the same Form.

Reply:
Yep, do you have any suggestions for what we can use for the group names instead?

Reply:
How about the hook name?

Reply:
That works. 2145b4d